recipes/xxhash/all/conandata.yml

```diff
@@ -8,3 +8,4 @@ sources:
 patches:
   "0.8.1":
     - patch_file: "patches/0.8.1-fix-cmakelists.patch"
+    - patch_file: "patches/0.8.1-fix-static-assert-link-error.patch"
```

Comment:
Since this is a new patch and you already wrote all the details in the PR it would be great to add them here too